resize/index.js

```diff
@@ -4,6 +4,18 @@ var AWS = require("aws-sdk");
 var im = require("gm").subClass({imageMagick: true});
 var s3 = new AWS.S3({signatureVersion: 'v4'});
 
+
+const DEFAULT_PICS_ORIGINAL_PATH = 'pics/original/';
+
+
+function getPicsOriginalPath() {
+  if (process.env.PICS_ORIGINAL_PATH) {
+    return process.env.PICS_ORIGINAL_PATH;
+  }
+
+  return DEFAULT_PICS_ORIGINAL_PATH;
```

Comment:
Does JS let you do something like this?

```suggestion
  if (process.env.PICS_ORIGINAL_PATH) {
    return process.env.PICS_ORIGINAL_PATH;
  }

  return process.env.PICS_ORIGINAL_PATH || DEFAULT_PICS_ORIGINAL_PATH;
```

Reply:
Yes, indeed it does - I think I had considered writing it like that, but didn't trust my JS knowledge enough, so I chickened out and wrote it more verbosely. Anyway, all tests still pass with it written as a one-liner, therefore it must be equivalent. I've committed your suggestion - thanks!